### PR TITLE
[CHK-706] presa in carico

### PR DESCRIPTION
**Pull request by ciuffagianluca**
When the result from nodo for a payment is a timeout and the payment is taken in charge, the outcome is ok, but the payment result is not still available, for a better user experience the result page should show a coherent message for this situation

#### List of Changes

Added new type of outcome
Evaluation of id status to check this OK peculiar result
Update translation

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/utils/transactions/TransactionResultUtil.ts [hunk 1]
```diff
@@ -19,6 +20,7 @@ export enum OutcomeEnum {
   INVALID_METHOD = "12",
   KO_RETRIABLE = "13",
   INVALID_SESSION = "14",
+  TAKE_IN_CHARGE = "15",
```
Comment (giovanniberti) [inline src/utils/transactions/TransactionResultUtil.ts:23]: [bikeshedding]
What about
```suggestion
  TAKEN_IN_CHARGE = "15",
```
?

File: src/utils/transactions/TransactionResultUtil.ts [hunk 2]
```diff
@@ -30,6 +32,7 @@ export enum ViewOutcomeEnum {
   INVALID_CARD = "7",
   CANCELED_BY_USER = "8",
   EXCESSIVE_AMOUNT = "10",
+  TAKE_IN_CHARGE = "15",
```
Comment (giovanniberti) [inline src/utils/transactions/TransactionResultUtil.ts:35]: (see above)

**Review by giovanniberti — APPROVED** [review]

**Comment by giovanniberti** [comment]
🚀

File: src/utils/transactions/TransactionResultUtil.ts (hunk 1 above)
Comment (ciuffagianluca) [reply src/utils/transactions/TransactionResultUtil.ts]: That's reasonable

File: src/utils/transactions/TransactionResultUtil.ts (hunk 2 above)
Comment (ciuffagianluca) [reply src/utils/transactions/TransactionResultUtil.ts]: That's reasonable

File: src/utils/transactions/TransactionResultUtil.ts [hunk 3]
```diff
@@ -243,5 +247,9 @@ export const getOutcomeFromAuthcodeAndIsDirectAcquirer = (
             )
           )
         )
-    )
+    ),
+    (outcomeEnum: OutcomeEnum) =>
+      outcomeEnum === OutcomeEnum.SUCCESS && idStatus === 14
```
Comment (infantesimone) [inline src/utils/transactions/TransactionResultUtil.ts:252]: are we sure that the state to consider is 14?

**Review by infantesimone — CHANGES_REQUESTED** [review]
this PR have to go into prod before ecommerce?

**Pull request closed without merging**